corehq/apps/users/views/__init__.py

```diff
@@ -940,18 +940,18 @@ def page_url(self):
 
     @property
     def page_context(self):
-        domain = Domain.get_by_name(self.request.domain)
+        domain_obj = Domain.get_by_name(self.request.domain)
         if self.request_form is None:
-            initial = {'domain': domain.name}
+            initial = {'domain': domain_obj.name}
             if self.request.user.is_authenticated:
                 initial.update({
                     'email': self.request.user.get_username(),
                     'full_name': self.request.user.get_full_name(),
                 })
             self.request_form = DomainRequestForm(initial=initial)
         return {
-            'domain': domain.name,
-            'domain_name': domain.display_name(),
+            'domain': domain_obj.name,
+            'domain_name': domain_obj.display_name(),
```

Comment:
This is lovely (although not made worse by this PR)... adding to the confusion about what `domain_name` means (I thought it normally was the same as `domain.name`).

Reply:
There's something of a convention of using `hr_name` (human-readable name). Updated in e109388cc4b